meta-balena-common/classes/image_types_resin.bbclass

```diff
@@ -310,7 +310,10 @@ IMAGE_CMD_resinos-img () {
     fi
 
     # Label what is not labeled
-    e2label ${RESIN_ROOT_FS} ${RESIN_ROOTA_FS_LABEL}
+    if [ "${RESIN_ROOT_FSTYPE}" = "hostapp-ext4" ]; then
+        e2label ${RESIN_ROOT_FS} ${RESIN_ROOTA_FS_LABEL}
+    fi
+
```

Comment:
I think it might be better to bbfatal for non ext4 filesystems here so that it is clear that the filesystem needs to be labelled. I expect things to break if the filesystem is not labelled.

Reply:
Yep, as I was working on support for adding another rootfs type, I thought the same thing. I've updated this branch with this change.